updater/versions/update-1.py

```diff
@@ -146,6 +183,17 @@ def write_crontab():
     with open(CRONTAB_PATH, "w") as out_file:
         out_file.write(CRONTAB)
 
+def write_avahi_ooniprobe():
+    with open(AVAHI_OONIPROBE_PATH, "w") as out_file:
+        out_file.write(AVAHI_OONIPROBE)
+
+def write_avahi_ssh():
+    with open(AVAHI_SSH_PATH, "w") as out_file:
+        out_file.write(AVAHI_SSH)
+
+def write_lepidopter_update_logrotate():
+    with open(LEPIDOPTER_UPDATE_LOGROTATE_PATH, "w") as out_file:
+        out_file.write(LEPIDOPTER_UPDATE_LOGROTATE)
```

Comment:
You need to also call these functions below to actually write the files.

It should happen after the deps are installed.